test/rules.js

```diff
@@ -12,3 +12,21 @@ test('badge - incorrect source', async t => {
 	t.is(result.ruleId, 'awesome/badge');
 	t.is(result.message, 'Incorrect badge source');
 });
+
+process.chdir('package-default');
```

Comment:
Do this inside the test and also reset it back to the previous value at the end of the test body.

Reply:
got it, will do